### PR TITLE
adding tlp v2 and new object icons

### PR DESCRIPTION
**Pull request by himynamesdave**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,18 +16,17 @@ If intelligence provider cares about structured machine-readable CTI, the report
...
-* [TLP tags and custom sidebar content renderer](https://traut.github.io/stixview/dist/demos/tags-and-custom-sidebar.html) — TLP marking definitions shown as tags and sidebar is rendered with provided function.
-* [Examples of various configuration settings](https://traut.github.io/stixview/dist/demos/misc.html)
 
+* [STIX2.1 demo](dist/demos/stix21-demo.html) — sample graph with all STIX 2.1 objects.
+* [Storyline](dist/demos/story.html) — multiple graphs per page, rendering selected entities from the same STIX bundle.
+* [Viewer](dist/demos/viewer.html) — graph viewer with custom controls.
+* [Drag-n-drop](dist/demos/drag-n-drop.html) — graph views with drag-n-drop enabled.
+* [Dark theme graph from inline data](dist/demos/load-data.html) — rendering graph from inline STIX2 bundle and custom styling.
+* [TLP tags and custom sidebar content renderer](dist/demos/tags-and-custom-sidebar.html) — TLP marking definitions shown as tags and sidebar is rendered with provided function.
+* [Examples of various configuration settings](dist/demos/misc.html)
```
Comment (traut) [inline README.md:29]: Relative links would point to the source files, not to the versions hosted at `traut.github.io/stixview`. The hosted versions are ready-to-use, which is valuable, I think

File: README.md [hunk 2]
```diff
@@ -65,86 +64,56 @@ Every holder element _must have_ one of `data-stix-gist-id`, `data-stix-url` or
...
+* `loadData(data)` — load STIX2 bundle from `data` JSON object and render on a graph.
+* `loadDataFromFile(file)` — load STIX2 bundle from `file` file object and render on a graph.
+* `loadDataFromUrl(url)` — load STIX2 bundle from remote URL and render on a graph.
+* `loadDataFromParamUrl(paramName)` — load STIX2 bundle from remote URL, configured in HTTP GET paramter with name in `paramName` and render on a graph.
+
+## Build
+
+```shell
+yarn build
+```
```
Comment (traut) [inline README.md:119]: nice, thank you for the markdown cleanup!

File: icons/bank-card.svg
Comment (traut) [inline icons/bank-card.svg:1]: is it a new official object type or a custom one?

File: icons/x-eclecticiq-hypothesis.svg [hunk 3]
```diff
@@ -1,4 +0,0 @@
-<svg width="200" height="200" viewBox="0 0 200 200" fill="none" xmlns="http://www.w3.org/2000/svg">
-<circle cx="100" cy="100" r="100" fill="#009688"/>
-<path d="M141.461 149H120.367V105.125H79.2344V149H58.1406V46.625H79.2344V88.1094H120.367V46.625H141.461V149Z" fill="white"/>
-</svg>
```
Comment (traut) [inline icons/x-eclecticiq-hypothesis.svg:4]: rip! 😄

File: src/icons.js [hunk 4]
```diff
@@ -30,34 +31,86 @@ const observedDataIcon = require(
...
 const toolIcon = require(
     '!svg-inline-loader?removeSVGTagAttrs=false!../icons/tool.svg');
 const vulnerabilityIcon = require(
     '!svg-inline-loader?removeSVGTagAttrs=false!../icons/vulnerability.svg');
-
-const hypothesisIcon = require(
-    '!svg-inline-loader?removeSVGTagAttrs=false!../icons/x-eclecticiq-hypothesis.svg');
+// DOGESEC custom SDO
+const weaknessIcon = require(
+    '!svg-inline-loader?removeSVGTagAttrs=false!../icons/weakness.svg');
```
Comment (traut) [inline src/icons.js:44]: ❤️

File: src/icons.js [hunk 5]
```diff
@@ -30,34 +31,86 @@ const observedDataIcon = require(
...
+const bankCardIcon = require(
+    '!svg-inline-loader?removeSVGTagAttrs=false!../icons/bank-card.svg');
+const cryptocurrencyTransactionIcon = require(
+    '!svg-inline-loader?removeSVGTagAttrs=false!../icons/cryptocurrency-transaction.svg');
+const cryptocurrencyWalletIcon = require(
+    '!svg-inline-loader?removeSVGTagAttrs=false!../icons/cryptocurrency-wallet.svg');
+const phoneNumberIcon = require(
+    '!svg-inline-loader?removeSVGTagAttrs=false!../icons/phone-number.svg');
+const userAgentIcon = require(
+    '!svg-inline-loader?removeSVGTagAttrs=false!../icons/user-agent.svg');
```
Comment (traut) [inline src/icons.js:99]: aha, clear now! all good

File: src/icons.js [hunk 6]
```diff
@@ -219,7 +377,6 @@ const unknownIconTmpl = `
 function getPlaceholderIcon(nodeType) {
     const template = _.template(unknownIconTmpl);
     const letter = nodeType.charAt(0).toUpperCase();
-    const color = cyberObsColors[nodeType] || '#B99435';
     return {
         color: color,
```
Comment (traut) [inline src/icons.js:381]: with the removal of line 222, this var `color` becomes undefined, I suspect

File: src/data.js [hunk 7]
```diff
@@ -31,26 +31,53 @@ const NODE_NAME_FIELDS = [
 
 function getTlpMarkings(bundle) {
     const tlpMarkings = {
+        // tlp v1
         'marking-definition--613f2e26-407d-48c7-9eca-b8e91df99dc9': {
             'id': 'marking-definition--613f2e26-407d-48c7-9eca-b8e91df99dc9',
-            'value': 'white',
+            'value': 'WHITE',
```
Comment (traut) [inline src/data.js:37]: should we follow a STIX2 tlp-marking dict and have these as lowercase?

**Review by traut — CHANGES_REQUESTED** [review]

File: src/icons.js (hunk 4 above)
Comment (himynamesdave) [reply src/icons.js:44]: ha, i removed it only b/c I'm 100% sure it's not used anymore. Plus, this is a deprecated custom object type

File: README.md (hunk 2 above)
Comment (himynamesdave) [reply README.md:119]: ha, was more a reminder for myself really

File: icons/bank-card.svg
Comment (himynamesdave) [reply icons/bank-card.svg:1]: it's a custom type I created. All of the new SCOs added in this commit

File: src/data.js (hunk 7 above)
Comment (himynamesdave) [reply src/data.js:37]: ah, good point

**Comment by himynamesdave** [comment]
Sweet! I've pushed an update based on your feedback

**Comment by traut** [comment]
merged in the scope of https://github.com/traut/stixview/pull/27

**Pull request closed without merging**